src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -331,3 +331,67 @@ def load_arguments(self, _):
+        c.argument('description', help="The description of the log alert rule.")
+        c.argument('tags', help="Space-separated tags in 'key[=value]' format.")
+
+    with self.argument_context('monitor log-alert create', arg_group='Schedule') as c:
+        c.argument('frequency', help="Frequency (in minutes) at which rule condition should be evaluated.")
+        c.argument('time_window')
+
+    with self.argument_context('monitor log-alert create', arg_group='Source') as c:
+        c.argument('data_source_id', help="The ARM resource id over which log search query is to be run.")
+        c.argument('alert_query')
+        c.argument('query_type')
+        c.argument('authorized_resources', nargs='+', help="Space separated list of ARM resource ids referred into query.")
+
+    with self.argument_context('monitor log-alert create', arg_group='Action') as c:
+        c.argument('severity', arg_type=get_enum_type(AlertSeverity), help="Severity of the alert.")
+        c.argument('throttling', help="Time (in minutes) for which Alerts should be throttled or suppressed.")
+        c.argument('threshold_operator', arg_type=get_enum_type(ConditionalOperator), help="Evaluation operation for rule.")
+        c.argument('threshold', help="Result or count threshold based on which rule should be triggered.")
+        c.argument('metric_column')
+        c.argument('metric_trigger_type', arg_type=get_enum_type(MetricTriggerType))
+        c.argument('metric_threshold_operator', arg_type=get_enum_type(ConditionalOperator))
+        c.argument('metric_threshold')
+        c.argument('action_group', nargs='+')
+        c.argument('email_subject', help="Custom subject override for all email ids in all Azure action group(s) associated with the alert rule.")
+
+    with self.argument_context('monitor log-alert update') as c:
+        c.argument('enabled', arg_type=get_three_state_flag(), help="Enable/disable log alert rule.")
+        c.argument('description', help="Update description for the log alert rule.")
+        c.argument('tags', help="Overwrite tags of an alert rule. Space-separated tags in 'key[=value]' format. Use "" to clear existing tags.")
+
+    with self.argument_context('monitor log-alert update', arg_group='Schedule') as c:
+        c.argument('frequency', help="Update frequency (in minutes) at which rule condition should be evaluated.")
+        c.argument('time_window', help="Update time window (in minutes) for which data needs to be fetched for query (should be greater than or equal to frequencyInMinutes).")
+
+    with self.argument_context('monitor log-alert update', arg_group='Source') as c:
+        c.argument('alert_query')
+        c.argument('reset_authorized_resources', action='store_true', help="Set authorized resources to null for alert rule.")
+        c.argument('add_authorized_resources', nargs='+', help="Add authorized resources to alert rule. Accepts space separated list of ARM resource ids.")
```

Comment:
hello. we prefer creating a subgroup like "monitor log-alert authorized-resources" and then you can add or remove them like "monitor log-alert authorized-resources add", "monitor log-alert authorized-resources remove" or "monitor log-alert authorized-resources list"

Reply:
"monitor log-alert authorized-resources reset"